blocks/header/header.js

```diff
@@ -430,14 +462,196 @@ function buildHamburger(nav, navSections) {
   return hamburger;
 }
 
+/**
+ * Build the hamburger icon element which will trigger the menu display in mobile mode
+ * @returns {HTMLDivElement} Breadcrumb header element
+ // * @param {String} title The content of the root breadcrumb element that should be displayed (page section)
+ // * @param {[Object]} links An array of link objects sorted in a parent -> child structure
+ */
+function buildBreadCrumbDesktop() {
+  const breadCrumb = div({ class: 'l3-nav-desktop' }, div({ class: 'l3-nav-desktop-menu-head' }));
+  breadCrumb.innerHTML = `
+    <div class="container section-container">
+      <div class="left-sec" tabindex="0">
+      </div>
+      <div class="right-sec">
+        <div class="right-sec-menu-option">
+          <a href="/unified-communications/pricing/" class="title-option" target="_self">
```

Comment:
If we ever expand beyond the scope of the unified-communications page this element will need to be dynamically populated.  But for now static HTML seems to suffice.